### PR TITLE
Cyp 667

### PR DESCRIPTION
**Pull request by HRanjan-11**

### CODE REVIEW

File: commands/utils/set_args.js [hunk 1]
```diff
@@ -434,6 +434,10 @@ function sync_args_from_cmd(args) {
       lt_config["run_settings"]["npm_via_tunnel"] = false;
     }
 
+    if ("max_duration" in args) {
+      lt_config["run_settings"]["max_duration"] = parseFloat(args["max_duration"]);
```
Comment (japneetlambdatest) [inline commands/utils/set_args.js:438]: duration should be int

File: commands/utils/validate.js [hunk 2]
```diff
@@ -51,6 +51,18 @@ module.exports = validate_config = function (lt_config, validation_configs) {
       reject("Error!! Parallels value not correct");
     }
 
+    //validate max duration
+    if ("max_duration" in lt_config["run_settings"]){
```
Comment (japneetlambdatest) [inline commands/utils/validate.js:55]: add validation that it should be a int and throw an error  if that fails , here error is thrown only in this condition
```parseFloat(max_duration) <2 || parseFloat(max_duration) > 240```

File: commands/utils/set_args.js (hunk 1 above)
Comment (HRanjan-11) [reply commands/utils/set_args.js:438]: it is kept float intentional, as we want float number also. we are converting it to floor value in liisa.

File: commands/utils/validate.js (hunk 2 above)
Comment (HRanjan-11) [reply commands/utils/validate.js:55]: it is kept float intentional, as we want float number also. we are converting it to floor value in liisa.

File: commands/utils/set_args.js (hunk 1 above)
Comment (japneetlambdatest) [reply commands/utils/set_args.js:438]: Can you also keep this as int

**Review by japneetlambdatest — APPROVED** [review]

**Pull request merged**